orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<ol><li>Make the title a third degree title (so with three hash signs and then $J_2$ Perturbations).</li><li>Move the reference to the very bottom of the notebook.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>